src/store/sqlite_store/notes.rs

```diff
@@ -527,6 +572,9 @@ fn parse_output_note_columns(
     let inclusion_proof: Option<String> = row.get(5)?;
     let serialized_note_script: Option<Vec<u8>> = row.get(6)?;
     let consumer_account_id: Option<i64> = row.get(7)?;
+    let created_at: u64 = row.get(8)?;
+    let submited_at: Option<u64> = row.get(9)?;
```

Comment:
```suggestion
    let submitted_at: Option<u64> = row.get(9)?;
```

There are other occurrences as well (like in `store.sql`)

Reply:
Done!